ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/ForkIdManager.java

```diff
@@ -121,50 +73,40 @@ public static ForkId readFrom(final RLPInput in) {
    * @return boolean (peer valid (true) or invalid (false))
    */
   boolean peerCheck(final ForkId forkId) {
-    return forkIDCheckers.stream().anyMatch(checker -> checker.check(forkId));
-  }
-
-  private static ForkIDChecker eip2124(
-      final Blockchain blockchain,
-      final long forkNext,
-      final List<ForkId> forkAndHashList,
-      final long highestKnownFork) {
-    return forkId -> {
-      if (forkId == null) {
-        return true; // Another method must be used to validate (i.e. genesis hash)
-      }
-      // Run the fork checksum validation rule set:
-      //   1. If local and remote FORK_CSUM matches, connect.
-      //        The two nodes are in the same fork state currently. They might know
-      //        of differing future forks, but that's not relevant until the fork
-      //        triggers (might be postponed, nodes might be updated to match).
-      //   2. If the remote FORK_CSUM is a subset of the local past forks and the
-      //      remote FORK_NEXT matches with the locally following fork block number,
-      //      connect.
-      //        Remote node is currently syncing. It might eventually diverge from
-      //        us, but at this current point in time we don't have enough information.
-      //   3. If the remote FORK_CSUM is a superset of the local past forks and can
-      //      be completed with locally known future forks, connect.
-      //        Local node is currently syncing. It might eventually diverge from
-      //        the remote, but at this current point in time we don't have enough
-      //        information.
-      //   4. Reject in all other cases.
-      if (isHashKnown(forkId.getHash(), forkAndHashList)) {
-        if (blockchain.getChainHeadBlockNumber() < forkNext) {
-          return true;
+    if (forkId == null) {
+      return true; // Another method must be used to validate (i.e. genesis hash)
+    }
+    // Run the fork checksum validation rule set:
+    //   1. If local and remote FORK_CSUM matches, connect.
```

Comment:
Is this code comment still accurate if you delete the second part?

Reply:
This is a full clean revert.  We can address stuff like this in another PR.